meta-lmp-base/recipes-security/optee/fio-se05x-cli_git.bb

```diff
@@ -0,0 +1,24 @@
+SUMMARY = "Foundries.io NXP SE05X Secure Element CLI"
+HOMEPAGE = "https://github.com/foundriesio/fio-se05x-cli"
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/BSD-3-Clause;md5=550794465ba0ec5312d6919e203a55f9"
+
+inherit pkgconfig
+
+DEPENDS = "optee-os-tadevkit optee-client openssl"
+
+SRC_URI = "git://github.com/foundriesio/fio-se05x-cli.git;protocol=https;branch=main"
+SRCREV = "79c4d152e822bedf98b1397cb25fe1c9f1c3fadb"
+
+S = "${WORKDIR}/git"
+
+TEEC_EXPORT = "${STAGING_DIR_HOST}${prefix}"
+EXTRA_OEMAKE = "TEEC_EXPORT=${TEEC_EXPORT} \
+		USERTA_EXPORT=${TEEC_EXPORT}/include/optee/export-user_ta"
```

Comment:
Is USERTA_EXPORT needed?

Reply:
just thought it would be a good idea to have util.h but not needed...will drop